nextroute/model_constraint_maximum_wait_stop.go

```diff
@@ -3,7 +3,9 @@ package nextroute
 import "github.com/nextmv-io/sdk/connect"
 
 // MaximumWaitStopConstraint is a constraint that limits the time a vehicle can
-// wait between two stops.
+// wait between two stops. Wait is defined as the time between starting a stop
+// and arriving at the location of the stop, [SolutionStop.StartValue()] -
```

Comment:
Should this be the other way around? I arrive first and then I start. This is just wording, but maybe that is a bit clearer.

Reply:
If you start at the time you arrive there is no waiting, starting later than arriving creates waiting time (at the gate for example).